nex2art/core/Upload.py

```diff
@@ -398,3 +405,13 @@ def incFileCount(self, fname, error=False):
             self.parent.prog.stepsmap['Artifacts'][4] += 1
             if error: self.parent.prog.stepsmap['Artifacts'][3] += 1
             self.parent.prog.refresh()
+
+    def isNexus3ChecksumFile(self, chapdir, fname):
+        blobpropertiesfile = os.path.join(chapdir, fname + '.properties')
+        blobproperties = open(blobpropertiesfile, 'r').read()
+        return ('.md5' in blobproperties) or ('.sha1' in blobproperties) or ('.sha256' in blobproperties)
+
+    def isNexus3NpmFolder(self, chapdir, fname):
+        blobpropertiesfile = os.path.join(chapdir, fname + '.properties')
+        blobproperties = open(blobpropertiesfile, 'r').read()
+        return (re.search('blob-name=@', blobproperties)) and (not re.search('blob-name=@.+\..*', blobproperties))
```

Comment:
This is actually already dealt with, in [this file](https://github.com/jfrog/nexus2artifactory/blob/master/nex2art/core/Npm.py). If you're getting this bug, the logic in that file might be too narrow, in which case you should probably modify it there instead of adding this here.

Reply:
Ah sure, the part which our folders are sneaking through is the self.checkContent(), as all the name properties in the blob bytes files are the full @scope/folder not just the folder. I've altered this to be `name in js['name']` from `js['name'] == name` in attempt to retain any existing cases covered by this.